frontend/src/types/AssessmentTypes.ts

```diff
@@ -1,10 +1,46 @@
+import { Grade } from "../APIClients/types/UserClientTypes";
+
 export type StatusProperty = "DRAFT" | "PUBLISHED" | "ARCHIVED" | "DELETED";
 
+export enum TestTypes {
+  Beginning = "BEGINNING",
+  End = "END",
+}
+
 export type AssessmentTypes = {
   status: StatusProperty;
   name: string;
-  grade: string;
-  type: "BEGINNING" | "END";
+  grade: Grade;
+  type: TestTypes;
   country: string;
   region: string;
 };
+
+export const gradeOptions = [
+  "Kindergarten",
+  "Grade 1",
+  "Grade 2",
+  "Grade 3",
+  "Grade 4",
+  "Grade 5",
+  "Grade 6",
+  "Grade 7",
+  "Grade 8",
+].map((value) => ({ value, label: value }));
```

Comment:
Should I be distinguishing from the value "K" and label "Kindergarten" as done in the Grade import type? And also should testTypeOptions be the enum values?

Reply:
i think we can just use the value "K"? where do you see "Kindergarten" being used?

Reply:
fixed!